js/index.js

```diff
@@ -0,0 +1,31 @@
+$(function() {
+
+  $('form').submit(switcher);
+
+    function switcher(event) {
+      event.preventDefault();
+      var inputinfo = $('#city-type').val();
+      // console.log(inputinfo);
+      if (inputinfo === "New York" || inputinfo === "NYC" || inputinfo === "New York City" || inputinfo === "nyc") {
+      console.log('empire');
+      $('body').css('background-image', 'url(../homework-04-citipix/images/nyc.jpg)')
```

Comment:
change the path of the image in `url(/path/to/file)`,   `images/nyc.jpg` should work 🔨